### PR TITLE
Consume operator config from openshift/api

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Depends on a re-bump with:
- https://github.com/openshift/api/pull/492
to get everything we need.

/hold

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
/hold remove

https://github.com/openshift/api/pull/492 merged, bumped, should be good to go.

**Comment by benjaminapetersen** [comment]
/hold cancel

always forget which one :)

**Comment by smarterclayton** [comment]
/hold

This isn’t passing and is in a retry loop

**Comment by spadgett** [comment]
/lgtm cancel

@benjaminapetersen the Dockerfile COPY command needs to be updated since the operator CRD is in a different folder

**Comment by benjaminapetersen** [comment]
/hold remove

This should fix, updated the `COPY` in the `Dockerfile`.

File: Dockerfile.rhel7 [hunk 1]
```diff
@@ -14,7 +14,8 @@ COPY --from=builder /tmp/build/console /usr/bin/console
 COPY manifests /manifests/
 
 # extensions manifests generated from openshift/api types
-COPY vendor/github.com/openshift/api/console/v1/*.yaml /manifests
+COPY vendor/github.com/openshift/api/console/v1/*.yaml /manifests/
+COPY vendor/github.com/openshift/api/operator/v1/0000_70_console-operator.crd.yaml /manifests/
```
Comment (spadgett) [inline Dockerfile.rhel7]: Is Dockerfile.rhel7 still used? I'm wondering if we can remove it and only maintain one Dockerfile.

Comment (benjaminapetersen) [reply Dockerfile.rhel7]: Good Q.  The `machine-`* operators, `cluster-version` operator, and `api-server` operators still have a `.rhel7` file.  I'm assuming its still in use.

**Comment by benjaminapetersen** [comment]
/retest

`timed out waiting for the condition`.

**Comment by benjaminapetersen** [comment]
@spadgett will do.

**Comment by spadgett** [comment]
@jhadvig FYI, this has the label selector change you added

**Comment by benjaminapetersen** [comment]
I think our new flake is a 100% merge blocker 
@jhadvig race to fix. 😢 😭 

error:
`timed out waiting for the condition`

**Comment by spadgett** [comment]
> I think our new flake is a 100% merge blocker

We should verify it's not something in this PR

**Pull request merged**